Mondee/View/TodayView/MondeeLevelView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MondeeLevelView.swift
+//  Mondee
+//
+//  Created by Seokmin on 2023/07/25.
+//
+
+import SwiftUI
+
+struct MondeeLevelView: View {
+    @ObservedObject var viewModel: TodayViewModel
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack {
+                Text("\(viewModel.currentLevel.name)")
+                    .font(.title)
+                    .fontWeight(.bold)
+                Spacer()
+            }
```

Comment:
```suggestion
        VStack(alignment:.leading, spacing: 0) {
            Text("\(viewModel.currentLevel.name)")
                .font(.title)
                .fontWeight(.bold)
```
alignment 사용하면 HStack 빼도 좋을 것 같아요!